app/refactor/views/p5/tad/components/table_3/row_2.scala.xml

```diff
@@ -11,7 +11,8 @@
                 <fo:table-body>
                     <fo:table-row>
                         <fo:table-cell border-top-style="solid" border-top-width="1pt" border-top-color="black" display-align="top" margin-left="2%" padding-bottom="138%" padding-top="5%">
-                            <fo:block font-weight="bold" font-size="5pt">CERTIFICATE BY COMPETENT AUTHORITIES</fo:block>
+                            <fo:block font-weight="bold" font-size="5pt">CERTIFICATION BY COMPETENT AUTHORITIES
+                            </fo:block>
```

Comment:
Should this not be Camel case?
CERTIFICATE BY COMPETENT AUTHORITIES: CERTIFICATION BYCOMPETENT AU-THORITIES .. but displayed “CERTIFICATE BY COMPETENT AUTHORITIES” - Action needed '  -->The label should display Certification by Competent Authorities'{}